doc/get_started/digi-embedded-yocto-2-0-connectcore-for-imx6ul-starter-kit-python.md

```diff
@@ -0,0 +1,220 @@
+---
+platform: digi embedded yocto 2.0
+device: connectcore 6 ul starter kit
+language: python
+---
+
+Run a simple PYTHON sample on ConnectCore 6 UL and ConnectCore 6 UL Starter board running Digi Embedded Yocto 2.0 (DEY2.0)
```

Comment:
Interesting.
When I ran on the Digi Embedded Linux running on a Digi XBee GW I had to cross compile the Python .so for ARM.
I do not see any cross compile steps in the instructions. Are the instructions saying to compile on the target?